package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@asyncapi/app-relations-discovery",
+  "version": "0.0.1",
+  "description": "A library that focuses on finding and analyzing the relationships between AsyncAPI Documents too later output consolidated information about the system architecture. Output format should be customizable and available in different formats like uml, mermaid.js or something similar.",
+  "main": "./lib/index.js",
+  "scripts": {
+    "test": "echo \"No test specified\"",
+    "build": "tsc",
+    "lint": "eslint --max-warnings 0 --config .eslintrc .",
+    "get:version": "echo $npm_package_version",
+    "get:name": "echo $npm_package_name",
+    "generate:assets": "npm run build",
```

As Jonas mentioned, you don't use TS so you should add placeholder for it:

 ```suggestion
    "generate:assets": "echo \"Nothing to generate\"",
```